template/web/wp-content/themes/studiometa/app/Repositories/PostTypeRepository.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Repository entity for retrieving post type objects.
+ *
+ * @package Studiometa
+ */
+
+namespace Studiometa\Repositories;
+
+/** Class */
+class PostTypeRepository extends Repository {
+	const POST_TYPES = array( 'post' ); // Main post types.
+
+	/**
+	 * Returns a chronological list of latest "Post" (articles) posts for a given
+	 * category. Default $limit is 10.
+	 *
+	 * @param string|array $slug    Slug.
+	 * @param integer      $limit   Number to return (optional).
+	 * @param array        $exclude Posts to exclude (optional).
+	 * @param integer      $paged   Enable pagination (optional).
+	 *
+	 * @return Repository
+	 */
+	public function posts_by_category_slug( $slug, $limit = 10, $exclude = array(), $paged = 0 ) {
+
+		// Set sane defaults so we don't do full table scans.
+		if ( $limit <= 0 || $limit > 100 ) {
+			$limit = 100;
+		}
+
+		// Note the + symbol. See https://codex.wordpress.org/Class_Reference/WP_Query#Category_Parameters.
```

Comment:
The URL is not valid anymore, the new one is https://developer.wordpress.org/reference/classes/wp_query/#category-parameters